### PR TITLE
Goodbye 2!

### PR DESCRIPTION
**Pull request by JasonTarka-D2L**
<a href='#crh-start'></a><a href='#crh-data-%7B%22processed%22%3A%20%5B%22https%3A//github.com/JasonTarka-D2L/ReviewNinja-Welcome/pull/5%23issuecomment-122391936%22%2C%20%22https%3A//github.com/JasonTarka-D2L/ReviewNinja-Welcome/pull/5%23issuecomment-122392026%22%2C%20%22https%3A//github.com/JasonTarka-D2L/ReviewNinja-Welcome/pull/5%23issuecomment-122392864%22%2C%20%22https%3A//github.com/JasonTarka-D2L/ReviewNinja-Welcome/pull/5%23issuecomment-122393033%22%2C%20%22https%3A//github.com/JasonTarka-D2L/ReviewNinja-Welcome/pull/5%23issuecomment-122404235%22%2C%20%22https%3A//github.com/JasonTarka-D2L/ReviewNinja-Welcome/pull/5%23discussion_r34924665%22%2C%20%22https%3A//github.com/JasonTarka-D2L/ReviewNinja-Welcome/pull/5%23discussion_r34924686%22%2C%20%22https%3A//github.com/JasonTarka-D2L/ReviewNinja-Welcome/pull/5%23discussion_r34924714%22%2C%20%22https%3A//github.com/JasonTarka-D2L/ReviewNinja-Welcome/pull/5%23discussion_r34924823%22%2C%20%22https%3A//github.com/JasonTarka-D2L/ReviewNinja-Welcome/pull/5%23discussion_r34924845%22%2C%20%22https%3A//github.com/JasonTarka-D2L/ReviewNinja-Welcome/pull/5%23discussion_r34924885%22%2C%20%22https%3A//github.com/JasonTarka-D2L/ReviewNinja-Welcome/pull/5%23discussion_r34925198%22%2C%20%22https%3A//github.com/JasonTarka-D2L/ReviewNinja-Welcome/pull/5%23discussion_r34925373%22%2C%20%22https%3A//github.com/JasonTarka-D2L/ReviewNinja-Welcome/pull/5%23discussion_r34925609%22%2C%20%22https%3A//github.com/JasonTarka-D2L/ReviewNinja-Welcome/pull/5%23discussion_r34925733%22%5D%2C%20%22comments%22%3A%20%7B%22General%20Comment%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/JasonTarka-D2L/ReviewNinja-Welcome/pull/5%23issuecomment-122391936%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22%3Ca%20href%3D%5C%22http%3A//app.review.ninja/JasonTarka-D2L/ReviewNinja-Welcome/pull/5%5C%22%20target%3D%5C%22_blank%5C%22%3E%3Cimg%20src%3D%5C%22http%3A//app.review.ninja/39269213/pull/5/badge%5C%22%20alt%3D%5C%22ReviewNinja%5C%22/%3E%22%2C%20%22created_at%22%3A%20%222015-07-17T19%3A47%3A30Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/8714728%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/review-ninja%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3A%2B1%3A%22%2C%20%22created_at%22%3A%20%222015-07-17T19%3A48%3A05Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/12836044%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/JasonTarka-D2L%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3Ashipit%3A%22%2C%20%22created_at%22%3A%20%222015-07-17T19%3A52%3A06Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/12836044%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/JasonTarka-D2L%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3Ashipit%3A%3Agun%3A%22%2C%20%22created_at%22%3A%20%222015-07-17T19%3A52%3A23Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/12836044%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/JasonTarka-D2L%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3Ashipit%3A%22%2C%20%22created_at%22%3A%20%222015-07-17T20%3A31%3A40Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/12836044%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/JasonTarka-D2L%22%7D%7D%5D%2C%20%22title%22%3A%20%22General%20Comment%22%7D%2C%20%22Pull%209c585fa0626ac7053a84e25746175375cd3e4707%20hello%205%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/JasonTarka-D2L/ReviewNinja-Welcome/pull/5%23discussion_r34924665%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22What%20is%20this%3F%22%2C%20%22created_at%22%3A%20%222015-07-17T19%3A48%3A26Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/12836044%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/JasonTarka-D2L%22%7D%7D%2C%20%7B%22body%22%3A%20%22Well%2C%20you%20know...%22%2C%20%22created_at%22%3A%20%222015-07-17T19%3A48%3A37Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/12836044%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/JasonTarka-D2L%22%7D%7D%2C%20%7B%22body%22%3A%20%22it%27s%20something%5Cr%5Cn%5Cr%5Cnthat%20has%20multiple%20lines%21%22%2C%20%22created_at%22%3A%20%222015-07-17T19%3A48%3A59Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/12836044%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/JasonTarka-D2L%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20hello%3AL2-9%22%7D%2C%20%22Pull%209c585fa0626ac7053a84e25746175375cd3e4707%20hello%207%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/JasonTarka-D2L/ReviewNinja-Welcome/pull/5%23discussion_r34924823%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22This%20is%20on%20a%20later%2C%20empty%2C%20line.%22%2C%20%22created_at%22%3A%20%222015-07-17T19%3A50%3A17Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/12836044%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/JasonTarka-D2L%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3A%2B1%3A%22%2C%20%22created_at%22%3A%20%222015-07-17T19%3A51%3A06Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/12836044%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/JasonTarka-D2L%22%7D%7D%2C%20%7B%22body%22%3A%20%22What%20if%20I%20comment%20again%3F%22%2C%20%22created_at%22%3A%20%222015-07-17T19%3A55%3A25Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/12836044%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/JasonTarka-D2L%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20hello%3AL2-9%22%7D%2C%20%22Pull%209c585fa0626ac7053a84e25746175375cd3e4707%20hello%206%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/JasonTarka-D2L/ReviewNinja-Welcome/pull/5%23discussion_r34924845%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22ANd%20a%20middle%20line%22%2C%20%22created_at%22%3A%20%222015-07-17T19%3A50%3A31Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/12836044%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/JasonTarka-D2L%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3A%2B1%3A%20%22%2C%20%22created_at%22%3A%20%222015-07-17T20%3A01%3A25Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/12836044%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/JasonTarka-D2L%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20hello%3AL2-9%22%7D%2C%20%22Pull%209c585fa0626ac7053a84e25746175375cd3e4707%20hello%203%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/JasonTarka-D2L/ReviewNinja-Welcome/pull/5%23discussion_r34925373%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22Test%20Comment%22%2C%20%22created_at%22%3A%20%222015-07-17T19%3A57%3A24Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/12845744%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/JasonGehl-D2L%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3A%2B1%3A%22%2C%20%22created_at%22%3A%20%222015-07-17T20%3A00%3A02Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/12845744%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/JasonGehl-D2L%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20hello%3AL2-9%22%7D%7D%2C%20%22approved%22%3A%20%7B%22https%3A//github.com/JasonTarka-D2L%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/12836044%3Fv%3D3%22%7D%7D%7D'></a>
<a href='https://www.codereviewhub.com/'></a>

<a href='https://github.com/JasonTarka-D2L'></a>

- [ ] <a href='#crh-comment-Pull 9c585fa0626ac7053a84e25746175375cd3e4707 hello 5'></a> &nbsp;<b><a href='https://github.com/JasonTarka-D2L/ReviewNinja-Welcome/pull/5#discussion_r34924665'>File: hello:L2-9</a></b>
- <a href='https://github.com/JasonTarka-D2L'></a> What is this?
- <a href='https://github.com/JasonTarka-D2L'></a> Well, you know...
- <a href='https://github.com/JasonTarka-D2L'></a> it's something
that has multiple lines!
- [ ] <a href='#crh-comment-Pull 9c585fa0626ac7053a84e25746175375cd3e4707 hello 7'></a> &nbsp;<b><a href='https://github.com/JasonTarka-D2L/ReviewNinja-Welcome/pull/5#discussion_r34924823'>File: hello:L2-9</a></b>
- <a href='https://github.com/JasonTarka-D2L'></a> This is on a later, empty, line.
- <a href='https://github.com/JasonTarka-D2L'></a> :+1:
- <a href='https://github.com/JasonTarka-D2L'></a> What if I comment again?
- [x] <a href='#crh-comment-General Comment'></a> &nbsp;<b><a href='https://github.com/JasonTarka-D2L/ReviewNinja-Welcome/pull/5#issuecomment-122391936'>General Comment</a></b>
- <a href='https://github.com/review-ninja'></a> <a href="http://app.review.ninja/JasonTarka-D2L/ReviewNinja-Welcome/pull/5" target="_blank">ReviewNinja
- <a href='https://github.com/JasonTarka-D2L'></a> :+1:
- <a href='https://github.com/JasonTarka-D2L'></a> :shipit:
- <a href='https://github.com/JasonTarka-D2L'></a> :shipit::gun:
- [x] <a href='#crh-comment-Pull 9c585fa0626ac7053a84e25746175375cd3e4707 hello 6'></a> &nbsp;<b><a href='https://github.com/JasonTarka-D2L/ReviewNinja-Welcome/pull/5#discussion_r34924845'>File: hello:L2-9</a></b>
- <a href='https://github.com/JasonTarka-D2L'></a> ANd a middle line
- [x] <a href='#crh-comment-Pull 9c585fa0626ac7053a84e25746175375cd3e4707 hello 3'></a> &nbsp;<b><a href='https://github.com/JasonTarka-D2L/ReviewNinja-Welcome/pull/5#discussion_r34925373'>File: hello:L2-9</a></b>
- <a href='https://github.com/JasonGehl-D2L'></a> Test Comment

<a href='https://www.codereviewhub.com/JasonTarka-D2L/ReviewNinja-Welcome/pull/5?mark_as_completed=1'></a>&nbsp;<a href='https://www.codereviewhub.com/JasonTarka-D2L/ReviewNinja-Welcome/pull/5?approve=1'></a>&nbsp;<a href='https://www.codereviewhub.com/JasonTarka-D2L/ReviewNinja-Welcome/pull/5?approve=0'></a>&nbsp;<a href='https://github.com/JasonTarka-D2L/ReviewNinja-Welcome/pull/5'></a>
<a href='#crh-end'></a>

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="http://app.review.ninja/JasonTarka-D2L/ReviewNinja-Welcome/pull/5" target="_blank">ReviewNinja

**Comment by JasonTarka-D2L** [comment]
This pull request is now managed with CodeReviewHub. <a href='https://github.com/JasonTarka-D2L/ReviewNinja-Welcome/pull/5'>Refresh</a>.<a href='#crh-update-none'></a>

**Comment by JasonTarka-D2L** [comment]
:+1:

File: hello [hunk 1]
```diff
@@ -2,4 +2,7 @@ This is a file
 with
 some
 
-Contents!
+Contents
```
Comment (JasonTarka-D2L) [inline hello]: What is this?

Comment (JasonTarka-D2L) [inline hello]: Well, you know...

Comment (JasonTarka-D2L) [inline hello]: it's something

that has multiple lines!

File: hello [hunk 2]
```diff
@@ -2,4 +2,7 @@ This is a file
 with
 some
 
-Contents!
+Contents
+
+
```
Comment (JasonTarka-D2L) [inline hello]: This is on a later, empty, line.

File: hello [hunk 3]
```diff
@@ -2,4 +2,7 @@ This is a file
 with
 some
 
-Contents!
+Contents
+
```
Comment (JasonTarka-D2L) [inline hello]: ANd a middle line

File: hello (hunk 2 above)
Comment (JasonTarka-D2L) [inline hello]: :+1:

**Comment by JasonTarka-D2L** [comment]
:shipit:

**Comment by JasonTarka-D2L** [comment]
:shipit::gun:

File: hello (hunk 2 above)
Comment (JasonTarka-D2L) [inline hello]: What if I comment again?

File: hello [hunk 4]
```diff
@@ -2,4 +2,7 @@ This is a file
 with
 some
```
Comment (JasonGehl-D2L) [inline hello]: Test Comment

Comment (JasonGehl-D2L) [inline hello]: :+1:

File: hello (hunk 3 above)
Comment (JasonTarka-D2L) [inline hello]: :+1:

**Comment by JasonTarka-D2L** [comment]
:shipit:

**Pull request closed without merging**